lib/generators/rolemodel/simple_form/templates/app/inputs/markdown_input.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class MarkdownInput < SimpleForm::Inputs::Base
```

Comment:
Is there more JS that is needed for this input? Seems like might be a bit specific for the base generator.

Reply:
Probably don't need this. I'll remove it.